src/backend/src/main/kotlin/org/icpclive/overlay/Routing.kt

```diff
@@ -28,13 +33,37 @@ fun Route.configureOverlayRouting() {
     flowEndpoint("/mainScreen") { DataBus.mainScreenFlow.await() }
     flowEndpoint("/contestInfo") { DataBus.currentContestInfoFlow() }
     flowEndpoint("/runs") { DataBus.contestStateFlow.await().map { it.runsAfterEvent.values.sortedBy { it.time } } }
+    webSocket("/teamRuns/{id}") {
+        val teamIdStr = call.parameters["id"]
+        if (teamIdStr.isNullOrBlank()) {
+            close(CloseReason(CloseReason.Codes.CANNOT_ACCEPT, "Invalid team id"))
+            return@webSocket
+        }
+        val teamId = teamIdStr.toTeamId()
+        val acceptedProblems = mutableSetOf<ProblemId>()
+        val allRuns = mutableMapOf<RunId, RunInfo>()
+        DataBus.contestStateFlow.await().first().runsAfterEvent.values
```

Comment:
Tbh, I don't like how it ended, but let's proceed with this, I'll rewrite this part later, not sure how exactly.